tag/remote/remote.go

```diff
@@ -73,7 +75,7 @@ func httpRequest(url, authorization, mode string) (*http.Response, error) {
 		return nil, err
 	}
 	if resp.StatusCode != 200 {
-		return nil, errors.New("Bad response status: " + resp.Status + " >> " + url)
+		return resp, errors.New("Bad response status: " + resp.Status + " >> " + url)
```

Comment:
This was the culprit of a "panic" failure, we should return resp, not `nil`, if we want to communicate request status to `httpRetriableRequest` wrapper function!